src/hotspot/cpu/s390/assembler_s390.inline.hpp

```diff
@@ -415,10 +415,13 @@ inline void Assembler::z_rosbg( Register r1, Register r2, int64_t spos3, int64_t
 }
 inline void Assembler::z_risbg( Register r1, Register r2, int64_t spos3, int64_t epos4, int64_t nrot5, bool zero_rest) { // Rotate then INS selected bits.  -- z196
   const int64_t len = 48;
```

Comment:
Changes are not necessary if `bool zero_rest` is used to control what happens to untouched destination bits.